.storybook/main.ts

```diff
@@ -1,12 +1,29 @@
-module.exports = {
+type Dir = {
+    from: string;
+    to: string;
+};
+
+type Main = {
+    stories: string[];
+    addons: string[];
+    staticDirs: Array<string | Dir>;
+    core: {
+        builder: string;
+    };
+    managerHead: (head: string) => string;
+};
+
+const main: Main = {
```

Comment:
Also for this there is a built in type
`import { StorybookConfig } from '@storybook/core-common';`